### PR TITLE
feat: add support for headphone/solo level setting

### LINKED ISSUE #26: list all capabilities in HELP.md
some are missing there, e.g. MUTE groups. Check what's missing and add it

### LINKED ISSUE #28: Add support for headphone/solo level setting
- [x] wait for support in connection lib (https://github.com/fmalcher/soundcraft-ui/issues/51)
- [x] add actions

### PR DESCRIPTION
**Pull request by fmalcher**
close #28
close #26

### CODE REVIEW

File: src/state.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class UiFeedbackState {
    * Get the current value for a given feedback ID
    * @param feedbackId Internal ID of the feedback
    */
-  get(feedbackId: string) {
+  get(feedbackId: string): void {
```
Comment (TheMrPoseidon) [inline src/state.ts:54]: Does this really need to be void?

Comment (fmalcher) [reply src/state.ts:54]: Ugh you're right, thanks for the review! Will change this tomorrow

**Review by TheMrPoseidon — APPROVED** [review]
Looks good, but currently I don't have the hardware here to test it

**Comment by fmalcher** [comment]
I tested it on a Ui24R so at least it works in general 😃

**Comment by krocheck** [comment]
Can you please remove `package-lock.json` and add to `.gitignore`

**Comment by fmalcher** [comment]
@krocheck Yes sure, I always forget that we shall use uarn over here 🙈

**Comment by fmalcher** [comment]
> Can you please remove package-lock.json and add to .gitignore

Done

**Comment by fmalcher** [comment]
@krocheck Can be merged now, please. 🙂

**Pull request merged**